templates/debug/scheduled-crons.php

```diff
@@ -13,6 +13,17 @@
 	$fs_options      = FS_Options::instance( WP_FS__ACCOUNTS_OPTION_NAME, true );
 	$scheduled_crons = array();
 
+	$title_tag             = isset($VARS['title_tag']) ? $VARS['title_tag'] : 'h1';
```

Comment:
@DanieleAlessandra Missing spaces. I recommend reviewing this and adding the relevant spacing accordingly:
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#whitespace